kpops/components/base_components/kubernetes_app.py

```diff
@@ -100,7 +100,13 @@ def to_helm_values(self) -> dict:
         return self.app.dict(by_alias=True, exclude_none=True, exclude_unset=True)
 
     def print_helm_diff(self, stdout: str) -> None:
-        current_release = self.helm.get_manifest(self.helm_release_name, self.namespace)
+        current_release = list(
+            self.helm.get_manifest(self.helm_release_name, self.namespace)
+        )
+        if current_release:
+            log.info(f"Helm release {self.helm_release_name} already exists")
+        else:
+            log.info(f"Helm release {self.helm_release_name} does not exist")
```

Comment:
I don't know if you are a fan of this syntax but:
```python
log.info(f"Helm release {self.helm_release_name} already exists") if current_release else log.info(f"Helm release {self.helm_release_name} does not exist")
```

Reply:
Show me the Python way

Reply:
But this gets formatted to
```
log.info(
            f"Helm release {self.helm_release_name} already exists"
        ) if current_release else log.info(
            f"Helm release {self.helm_release_name} does not exist"
        )
```
so has even more lines

Reply:
@disrupted I would like to know your opinion on this 🐍

Reply:
Okay then lets revert it

Reply:
I am fine with the original. The only shorter option I can think of would be

```python
log.info(f"Helm release {self.helm_release_name} {'already exists' if current_release else 'does not exist'}")
```

Reply:
Let's stick with the good old if/else statement